src/main/java/org/jahia/modules/htmlfiltering/graphql/models/GqlRichTextConfig.java

```diff
@@ -24,7 +24,7 @@
 import java.util.List;
 import java.util.Set;
 
-@GraphQLDescription("Model for richtext configuration")
+@GraphQLDescription("Model for HTML Filtering")
 public class GqlRichTextConfig implements RichTextConfigInterface {
```

Comment:
```suggestion
public class GqlHTMLFilteringConfig implements RichTextConfigInterface {
```
But this needs to be done everywhere

Reply:
I guess I missed that one and the related interface